SimPEG/EM/Static/IP/ProblemIP.py

```diff
@@ -47,68 +54,111 @@ def fields(self, m):
             u = self.Ainv * RHS
             Srcs = self.survey.srcList
             self.f[Srcs, self._solutionType] = u
-        return self.f
 
-    def Jvec(self, m, v, f=None):
+    def getJ(self, f=None):
+        """
+            Generate Full sensitivity matrix
+        """
 
-        if f is None:
-            f = self.fields(m)
+        print (">> Compute Sensitivity matrix")
 
-        self.model = m
+        if self.f is None:
+            self.fieldsdc()
 
-        Jv = []
-        A = self.getA()
+        Jt = []
 
-        for src in self.survey.srcList:
-            u_src = f[src, self._solutionType] # solution vector
-            dA_dm_v = self.getADeriv(u_src, v)
-            dRHS_dm_v = self.getRHSDeriv(src, v)
-            du_dm_v = self.Ainv * ( - dA_dm_v + dRHS_dm_v )
+        AT = self.getA()
 
+        for isrc, src in enumerate(self.survey.srcList):
+            sys.stdout.write(("\r %d / %d")%(isrc, self.survey.nSrc))
```

Comment:
I think we should look into a logging package instead of using `sys.stdout` https://docs.python.org/3/library/logging.html